src/aap_eda/core/enums.py

```diff
@@ -24,6 +24,18 @@ def choices(cls):
     def values(cls):
         return tuple(e.value for e in cls)
 
+    def __eq__(self, other):
```

Comment:
- What if the enum is not used with strings? (unlikely but possible TBH) Somehow I feel would be better and simpler if our string enums inherits explicitly also from str. Equivalent to the new [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum)
eg:
```
class RestartPolicy(str, DjangoEnum):
```
- If we go with this approach should the class be renamed to `DjangoStrEnum`?

Reply:
An excellent suggestion.

Reply:
Ugh.  StrEnum is new in 3.11.

Reply:
yeah, that's why I suggested inherit from str where we want to deal with strEnums:
```
class RestartPolicy(str, DjangoEnum):
```